lib/worker/entity.js

```diff
@@ -7,10 +7,24 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
-// Pass submission def id in event
-const createEntityFromSubmission = ({ Entities }, event) => {
-  if (event.details.reviewState === 'approved' && event.details.submissionDefId)
-    return Entities.processSubmissionDef(event.details.submissionDefId);
+const createEntityFromSubmission = ({ Audits, Entities }, event) => {
+  if (event.details.reviewState === 'approved' && event.details.submissionDefId) {
+    return Entities.processSubmissionDef(event.details.submissionDefId)
+      .then((entity) => {
+        // TODO: figure out target for this audit. Could be the form of the
+        // original event or it could be the entity's dataset.
+        Audits.log({ id: event.actorId }, 'entity.create', null,
+          { entityUuid: entity.uuid,
+            submissionId: event.details.submissionId,
+            submissionDefId: event.details.submissionDefId });
+      })
+      .catch((problem) => {
+        Audits.log({ id: event.actorId }, 'entity.create.error', null,
```

Comment:
Similarly here, I think there'll need to be a new `if` in `actionCondition()` for entities, in order to support /v1/audits?action=entity. We don't show submission audit log entries in the Frontend audit log, and I think we'll probably want to exclude entity audit log entries as well. To accomplish that, these new actions will need to be added to the `nonverbose` condition, which Frontend requests by default.